minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -51,16 +53,21 @@
 
   private final ClientPool mClientPool = new ClientPool(mClientSupplier);
 
-  private MasterProcess mMasterProcess;
+  private AlluxioMasterProcess mMasterProcess;
   private Thread mMasterThread;
 
   private AlluxioSecondaryMaster mSecondaryMaster;
   private Thread mSecondaryMasterThread;
 
-  private LocalAlluxioMaster() throws IOException {
+  private ServerSocket mRpcBindSocket;
```

Comment:
these are `@Nullable`

Reply:
Are you suggesting to annotate the fields with `@Nullable`?